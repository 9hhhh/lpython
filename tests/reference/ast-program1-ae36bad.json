{
    "basename": "ast-program1-ae36bad",
    "cmd": "lfortran --show-ast {infile} -o {outfile}",
    "infile": "tests/program1.f90",
    "infile_hash": "7216936b0438627f7192ce2585664640fe73d45f9f29a613a65b3244",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-program1-ae36bad.stdout",
    "stdout_hash": "ab153df669659e280a38934d4180c18b89e4bf9f88e7bc617e205b3c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}