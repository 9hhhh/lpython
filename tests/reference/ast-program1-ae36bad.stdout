(TranslationUnit [(Program program1 [] [(Declaration [(i "integer" [] [] ())])] [(= i 5)] [])])
